src/Sarif.Viewer.VisualStudio/Fixes/EditActionPreviewProvider.cs

```diff
@@ -92,10 +94,11 @@ public class EditActionPreviewProvider : IPreviewProvider
                 Separator,
                 changedLineSpans.ToArray());
 
-            return this.CreateNewDifferenceViewerAsync(originalProjectionBuffer, newProjectionBuffer, description, additionalContent);
+            return this.CreateNewDifferenceViewerAsync(errorListItem, originalProjectionBuffer, newProjectionBuffer, description, additionalContent);
         }
 
         private async Task<object> CreateNewDifferenceViewerAsync(
```

Comment:
>object [](http://example.com/codeflow?start=27&length=6)

do we need to return object?

Reply:
I think this is because interface ISuggestedAction from Microsoft.VisualStudio.Language.Intellisense defines a method 
Task<object> GetPreviewAsync(CancellationToken cancellationToken); 

This preview provider just returns same type:

https://github.com/microsoft/sarif-visualstudio-extension/blob/fd4625d60eb3fd446e3f104450ca32c16f01121f/src/Sarif.Viewer.VisualStudio/Fixes/FixSuggestedAction.cs#L88L97